concent_api/core/subtask_helpers.py

```diff
@@ -146,11 +146,15 @@ def update_subtask_state(
     subtask.save()
 
 
-def verify_message_subtask_results_accepted(subtask_results_accepted_list: dict) -> bool:
+def verify_message_subtask_results_accepted(subtask_results_accepted_list: list) -> bool:
     """
     function verify if all requestor public key and ethereum public key
     in subtask_reesults_accepted_list are the same
     """
-    verify_public_key           = len(set(subtask_results_accepted.task_to_compute.requestor_public_key             for subtask_results_accepted in subtask_results_accepted_list)) == 1
-    verify_ethereum_public_key  = len(set(subtask_results_accepted.task_to_compute.requestor_ethereum_public_key    for subtask_results_accepted in subtask_results_accepted_list)) == 1
-    return bool(verify_public_key is True and verify_ethereum_public_key is True)
+    is_public_key_equal = len(
+        set(subtask_results_accepted.task_to_compute.requestor_public_key for subtask_results_accepted in subtask_results_accepted_list)) == 1
+    is_ethereum_address_key_equal = len(
+        set(subtask_results_accepted.task_to_compute.requestor_ethereum_address for subtask_results_accepted in subtask_results_accepted_list)) == 1
+    is_ethereum_public_key_equal = len(
+        set(subtask_results_accepted.task_to_compute.requestor_ethereum_public_key for subtask_results_accepted in subtask_results_accepted_list)) == 1
```

Comment:
Please fix the style here. This should be:

``` python
is_ethereum_public_key_equal = len(
    set(subtask_results_accepted.task_to_compute.requestor_ethereum_public_key for subtask_results_accepted in subtask_results_accepted_list)
) == 1
```

But I'd just use a helper variable:
``` python
public_keys          = set(subtask_results_accepted.task_to_compute.requestor_public_key          for subtask_results_accepted in subtask_results_accepted_list)
ethereum_addresses   = set(subtask_results_accepted.task_to_compute.requestor_ethereum_address    for subtask_results_accepted in subtask_results_accepted_list)
ethereum_public_keys = set(subtask_results_accepted.task_to_compute.requestor_ethereum_public_key for subtask_results_accepted in subtask_results_accepted_list)

is_public_key_equal           = (len(public_keys)          == 1)
is_ethereum_address_key_equal = (len(ethereum_addresses)   == 1)
is_ethereum_public_key_equal  = (len(ethereum_public_keys) == 1)
```

Reply:
done, and Your suggestion is short and easy to read.

Reply:
It still does not follow our styie. If you insist on having the parameter on a separate line then the closing parenthesis must be on a separate line too:

``` python
unique_public_keys = set(
    subtask_results_accepted.task_to_compute.requestor_public_key for subtask_results_accepted in subtask_results_accepted_list
)
```

It would be better to just keep it in a single line.

Reply:
We do not have hard line length limit. Just a suggestion to limit length to 120. The goal is readability. Sometimes it's more readable to just leave a long line as is. Especially when there are several similar long lines next to each other.

Reply:
ok :) it's done